adafruit_thermal_printer/thermal_printer.py

```diff
@@ -386,7 +387,8 @@ def set_defaults(self):
         self.size = SIZE_SMALL
         self.underline = None
         self.inverse = False
-        self.upside_down = False
+        self.upside_down = False  # wg. dokumentacji powinno działać ale nie działa
+        self.up_down_mode = True  # zamiast powyższego zaiplementowałem to   <<
```

Comment:
Mind commenting these in English? Most folks we work with (including myself) speak English.

Reply:
Yes. Sorry. I worked at this part like 2 months ago and didn't pay attention to it when I was preparing for commit